### PR TITLE
Flow solidus demo site - frontend flow priceing, rake jobs to flowcommerce gem

### PR DESCRIPTION
**Pull request by dux**

### CODE REVIEW

File: lib/tasks/flow.rake [hunk 1]
```diff
@@ -63,24 +64,24 @@ namespace :flow do
...
   end
 
   desc 'get catalog items'
   task get_catalog_items: :environment do
     # https://api.flow.io/reference/countries
     # https://docs.flow.io/#/module/localization/resource/experiences
 
-    for country_id in Flow.country_codes
+    client        = FlowCommerce.instance
+    api_data      = client.experiences.get(ENV.fetch('FLOW_ORG'))
```
Comment (mbryzek) [inline lib/tasks/flow.rake]: we need to loop here to ensure we go through all experiences

File: lib/tasks/flow.rake [hunk 2]
```diff
@@ -63,24 +64,24 @@ namespace :flow do
...
+    country_codes = api_data.map(&:country).map(&:downcase)
```
Comment (mbryzek) [inline lib/tasks/flow.rake]: we should use remove country_codes - and use "experience" below to get items for a given experience

File: SOLIDUS_FLOW.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Integration Slidus with flow - notes
```
Comment (mikeclimbrock) [inline SOLIDUS_FLOW.md]: `Slidus` => `Solidus`

File: SOLIDUS_FLOW.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+* Migrate database and create cache tables in your shop
+* replace all calls in frontend templates from "link_to_cart" to "flow_link_to_cart"
+* replace all calls in frontend templates from "order.display_item_total.to_html" to "flow_cart_total"
+
+
+### flow.io requirements
+
+* create catalog
+* create experiences in flow.io
+* create at least one distribution center in every expericence
```
Comment (mikeclimbrock) [inline SOLIDUS_FLOW.md]: `expericence` => `experience`

File: SOLIDUS_FLOW.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+* defined flow ENV variables, FLOW_API_KEY and FLOW_ORG
+
+
+### Sync your catalog to and from Flow.io
+
+Products in solidus are automaticly refreshed from flow,
```
Comment (mikeclimbrock) [inline SOLIDUS_FLOW.md]: `automaticly` => `automatically`

File: app/helpers/flow_helper.rb [hunk 6]
```diff
@@ -15,4 +15,48 @@ def flow_price(product)
...
+      css_class = :empty
+    else
+      text = '%s: (%s) <span class="amount">%s</span>' % [text, simple_current_order.item_count, flow_cart_total]
+      css_class = :full
+    end
+
+    link_to text.html_safe, spree.cart_path, class: 'cart-info %s' % css_class
+  end
+
+  # get flow item from line item and shows harmonized price
```
Comment (mikeclimbrock) [inline app/helpers/flow_helper.rb]: `harmonized` => 'localized`

Harmonization is really a separate process (https://docs.flow.io/#/module/landed cost/resource/harmonized-items) than retrieving the localized prices for items given an experience

File: lib/tasks/flow.rake (hunk 1 above)
Comment (dux) [inline lib/tasks/flow.rake]: we loop. line #84 is

> for country_id in country_codes

but I will convert it to block

> country_codes.each do |country_id|

for looping is old habit, ah

File: lib/tasks/flow.rake (hunk 2 above)
Comment (dux) [inline lib/tasks/flow.rake]: ok, will create variable country_id inside each block to make it more readable

**Pull request merged**